src/FormStartAdornment/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import { useFormControlInternalContext } from "../shared/FormControlContext";
+
+/**
+ * Component to render a start adornment.
+ *
+ * This is intended to be rendered below `<FormControl>`. If this is rendered on
+ * it's own; it will render `children` without any modification.
+ */
+export function FormStartAdornment({
+  children,
+}: {
+  children: React.ReactNode;
+}): React.ReactNode {
+  const { setStartAdornment } = useFormControlInternalContext();
+
+  React.useLayoutEffect(() => {
+    setStartAdornment?.(children);
+  }, [children, setStartAdornment]);
+
+  return setStartAdornment ? null : children;
```

Comment:
(fleshing out what I meant above)
```suggestion
  return (
    <div
      css={{
        position: "absolute",
        display: "inline-flex",
        left: 12,
        top: "50%",
        transform: "translateY(-50%)",
      }}
    >
      {children}
    </div>
  );
```

Reply:
Done!